### PR TITLE
feat(ui, models): Display more model metrics

### PR DESCRIPTION
**Pull request by dwesolow**
## Description

image
## Related Issues

Part 3 of #961

### CODE REVIEW

**Comment by MarkRedeman** [comment]
Few quick notes (most already applicable to what we have on main, but would be nice if we can improve this here):
1. Let's make all graphs x-axis start at `step = 0`, even if the data doesn't contain this point
2. Let's change the x scale so that the values are rounded, so instead of 49, 999, 1949 etc we would have 0, 1000, 2000
3. Change the graph colors: training loss in green, validation loss in red
4. Add a tooltip on hover showing the exact x and y values
5. (follow up?) when hovering inside of "training loss" show a vertical line in validation loss and learning rate and a tooltip of the graphs values at that point.
6. (follow up?) allow users to select an area in the chart and zoom into that, double click to reset zoom and/or add a button to reset zoom
7. (later, need to experiment with this) I think it could be nice if we show a vertical line or some symbol at every "epoch" tat's finished
8. (later, need to experiment with this) with Albert's fix from #1002 we no longer save the latest checkpoint, instead we store the checkpoint associated to the epoch where validation loss was lowest. In #895 we also want to allow users to store `n` checkpoints, in this case we can add symbols/anontations in our graphs to highlight those points

**Comment by dwesolow** [comment]
> Few quick notes (most already applicable to what we have on main, but would be nice if we can improve this here):
> 
> 1. Let's make all graphs x-axis start at `step = 0`, even if the data doesn't contain this point
> 2. Let's change the x scale so that the values are rounded, so instead of 49, 999, 1949 etc we would have 0, 1000, 2000
> 3. Change the graph colors: training loss in green, validation loss in red
> 4. Add a tooltip on hover showing the exact x and y values
> 5. (follow up?) when hovering inside of "training loss" show a vertical line in validation loss and learning rate and a tooltip of the graphs values at that point.
> 6. (follow up?) allow users to select an area in the chart and zoom into that, double click to reset zoom and/or add a button to reset zoom
> 7. (later, need to experiment with this) I think it could be nice if we show a vertical line or some symbol at every "epoch" tat's finished
> 8. (later, need to experiment with this) with Albert's fix from [fix(training): Keep best checkpoint instead of last #1002](https://github.com/open-edge-platform/physical-ai-studio/pull/1002) we no longer save the latest checkpoint, instead we store the checkpoint associated to the epoch where validation loss was lowest. In [Extend model training options #895](https://github.com/open-edge-platform/physical-ai-studio/issues/895) we also want to allow users to store `n` checkpoints, in this case we can add symbols/anontations in our graphs to highlight those points

Thanks for the feedback! In this PR points 1-4 are resolved.

**Review by MarkRedeman — DISMISSED** [review]

**Review by MarkRedeman — APPROVED** [review]
